package.json

```diff
@@ -30,6 +30,7 @@
     "@sveltejs/eslint-config": "^8.3.3",
     "@svitejs/changesets-changelog-github-compact": "^1.1.0",
     "@types/node": "^20.11.5",
+    "@types/picomatch": "^4.0.2",
```

Comment:
beats me why we suddenly need this to solve some typechecking errors, but that's the javascript ecosystem for you